server/hedley/modules/custom/hedley_fields/hedley_fields.features.field_base.inc

```diff
@@ -963,12 +963,13 @@ function hedley_fields_field_default_field_bases() {
     'module' => 'list',
     'settings' => array(
       'allowed_values' => array(
-        'add to calendar icon' => 'add to calendar icon',
-        'alarm icon' => 'alarm icon',
-        'bug icon' => 'bug icon',
-        'paw icon' => 'paw icon',
-        'find icon' => 'find icon',
-        'handshake icon' => 'handshake icon',
+        'calendar' => 'Add to calendar',
```

Comment:
That's the easiest solution I've found, of course we can keep an array with the icons values and preprocess it, just took a shortcut in here.